pkg/client/auth/exec/authenticator.go

```diff
@@ -0,0 +1,98 @@
+package exec
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	grpc_credentials "google.golang.org/grpc/credentials"
```

Comment:
Looks like this is out of order, not sure why it's not being picked up by `goimports`

Reply:
done